app/views/layouts/application.html.erb

```diff
@@ -6,24 +6,18 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
+    <%= javascript_pack_tag "toastr", "data-turbolinks-track": "reload" %>
```

Comment:
import vào application.js đi e, a thấy e dùng yarn add vào package.json mà sao vẫn require nhiều cái js trùng nhau thế. Nhìn thư mục javascript/packs ghê ghê :))

Reply:
đọc cái này để hiểu thêm 1 ít nè :)) https://robrace.dev/integrating-bootstrap-4-in-rails-6/